src/platform/packages/shared/kbn-test/src/functional_tests/run_tests/run_tests.ts

```diff
@@ -127,6 +127,39 @@ export async function runTests(log: ToolingLog, options: RunTestsOptions) {
             ],
           });
 
+          const startRemoteKibana = config.get('kbnTestServer.startRemoteKibana');
+
+          if (startRemoteKibana) {
```

Comment:
just looking at the code above, if `startRemoteKibana == true` then `runKibanaServer` will be called twice - once as it was, and once with this remote config. Is it intentional? aren't these two supposed to be exclusive? (i.e.: remote kibana vs local kibana?)